src/examples/java/com/devcycle/examples/LocalExample.java

```diff
@@ -22,8 +23,13 @@ public static void main(String[] args) throws InterruptedException {
         // The default value can be of type string, boolean, number, or JSON
         Boolean defaultValue = false;
 
-        DevCycleLocalOptions options = DevCycleLocalOptions.builder().configPollingIntervalMs(60000)
-                .disableAutomaticEventLogging(false).disableCustomEventLogging(false).build();
+        DevCycleLocalOptions options = DevCycleLocalOptions.builder()
+                .configPollingIntervalMS(60000)
+                .disableAutomaticEventLogging(false)
+                .customLogger(new SimpleDevCycleLogger(SimpleDevCycleLogger.Level.DEBUG))
+                .disableCustomEventLogging(false)
+                .enableBetaRealtimeUpdates(true)
+                .build();
```

Comment:
are half of these needed? won't it keep events by default?

Reply:
This is just reformatted from what it was before

Reply:
can we cleanup the ones here that are just using the default values?